generated/nidaqmx/_base_interpreter.py

```diff
@@ -65,8 +65,8 @@ def cfg_anlg_edge_start_trig(
 
     @abc.abstractmethod
     def cfg_anlg_multi_edge_ref_trig(
-            self, task, trigger_sources, trigger_slope_array,
-            trigger_level_array, pretrigger_samples):
+            self, task, trigger_sources, pretrigger_samples,
+            trigger_slope_array, trigger_level_array):
```

Comment:
It's kind of weird that the position of pretrigger_samples in the parameter list changed, but I think it's consistent between all 3 interpreter classes and the caller.

@zhindes Do the changes you proposed affect this too?

Reply:
I think is because of what I found, yes. The C interface is this:

```C
 int32 __CFUNC     DAQmxCfgAnlgMultiEdgeRefTrig   (TaskHandle taskHandle, const char triggerSources[], const int32 triggerSlopeArray[], const float64 triggerLevelArray[], uInt32 pretriggerSamples, uInt32 arraySize); 
```

Notice that the arraySize doesn't follow the "cluster" of arrays. It looks like all of this Python codegen actually handles that correctly (the code _library_interpreter.py looks correct).

```Python
        cfunc = lib_importer.windll.DAQmxCfgAnlgMultiEdgeRefTrig
        if cfunc.argtypes is None:
            with cfunc.arglock:
                if cfunc.argtypes is None:
                    cfunc.argtypes = [
                        lib_importer.task_handle, ctypes_byte_str,
                        wrapped_ndpointer(dtype=numpy.int32, flags=('C')),
                        wrapped_ndpointer(dtype=numpy.float64, flags=('C')),
                        ctypes.c_uint32, ctypes.c_uint]

        error_code = cfunc(
            task, trigger_sources, trigger_slope_array, trigger_level_array,
            pretrigger_samples, len(trigger_level_array))
```

However, I think it caused this oddity in the codegen of the interpreter signatures. I will be adding `DAQmxCfgAlgMultiEdgeRefTrigEx` which is more sane. I think if we used that, we could make this a bit less confusing, but it doesn't look like we need to. I'll bubble my changes up through scrapigen and here just to make sure I didn't break anything unexpectedly.